Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -276,7 +276,7 @@
   - type: RevolutionaryRule
   - type: SpecialLobbyContent
     music: "/Audio/_Starlight/Music/Lobby/Liberation.ogg"
-    background: "/Textures/_Starlight/LobbyScreens/rev-clown.webp"
+    background: RevClown
```

Comment:
id be inclined to scream sl comments, but nothing else in this file does it so i dont think it matters